syncd/Syncd.cpp

```diff
@@ -962,7 +966,47 @@ sai_status_t Syncd::processBulkQuadEventInInitViewMode(
             }
 
         case SAI_COMMON_API_BULK_GET:
-            SWSS_LOG_THROW("GET bulk api is not implemented in init view mode, FIXME");
+            if (info->isnonobjectid)
+            {
+                /*
+                * Those objects are user created, so if user created ROUTE he
+                * passed some attributes, there is no sense to support GET
+                * since user explicitly know what attributes were set, similar
+                * for other non object id types.
+                */
```

Comment:
but thoe objects may have readonly attributes that can dynamically change and user may want to query those, for example SAI_ROUTE_ENTRY_ATTR_IP_ADDR_FAMILY

Reply:
This comment is inherited from processQuadInInitViewModeGet(). I do not want to make the behaviour incosistent between get vs. get bulk thus I support only objects

Reply:
@kcudnik can be resolved?